### PR TITLE
@broskoski => Proper sale_message display on artwork page + inquiry update

### PR DESCRIPTION
**Pull request by mzikherman**
This is now all that's needed to display the appropriate sale messaging on the artwork page (cc @xtina-starr this may help you for the microgravity artwork page).

Still to do:
- move the pre-filled inquiry text (based on availability) up to Metaphysics (for use in multiple clients)
- update pre-filled inquiry text in Force to use that

This PR is probably mergeable w/o the above steps, but no rush- let's hold this open for those issues as well.

### CODE REVIEW

File: apps/artwork/components/commercial/templates/price.jade [hunk 1]
```diff
@@ -1,10 +1,11 @@
+- var showPriceLabel = artwork.price && artwork.availability !== 'sold'
```
Comment (mzikherman) [inline apps/artwork/components/commercial/templates/price.jade]: Basically we just render `sale_message` directly now, except in the case of the `sale_message` containing some sort of pricing information (and not for sold works). In that case, we want to still just render the `sale_message`, but format it differently w/ some pricing labels/tooltips.

screen shot 2017-02-01 at 2 32 13 pm

**Comment by broskoski** [comment]
Nice and tidy! Sounds good, holding.

File: apps/artwork/components/commercial/templates/price.jade [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 if artwork.sale_message
   .artwork-commercial__container
     .artwork-commercial__sale-message
-      if artwork.price
+      if showPriceLabel
         .artwork-commercial__price-label Price
         .artwork-commerical__price
-          | #{artwork.price}
+          | #{artwork.sale_message}
```
Comment (mzikherman) [inline apps/artwork/components/commercial/templates/price.jade]: We still just directly render `sale_message` tho, so the conditional here is just for UI-specific DOM differences. I think the goal of centralizing and directly rendering `sale_message` is still accomplished.

**Comment by mzikherman** [comment]
Ok, I think this mergable on 💚 , it uses Metaphysics, plus i updated an older version of it as well.

**Comment by mzikherman** [comment]
Closes https://github.com/artsy/force/issues/790 ( + microgravity proxy)

File: components/contact/default_message.coffee [hunk 3]
```diff
@@ -3,5 +3,9 @@ module.exports = (artwork, partner) ->
     'Hello, I am interested in placing a bid on this work. ' +
```
Comment (mzikherman) [inline components/contact/default_message.coffee]: This is for the stand-alone `/inquiry/:artwork_id` route (serving Microgravity proxy).

**Comment by mzikherman** [comment]
💚

File: components/contact/default_message.coffee (hunk 3 above)
Comment (broskoski) [inline components/contact/default_message.coffee]: 👍

**Comment by broskoski** [comment]
Very nice.

**Pull request merged**